### PR TITLE
feat(text-field): update to v1.1.1

### PR DESCRIPTION
**Pull request by TroyTae**

### CODE REVIEW

File: packages/text-field/icon/index.tsx [hunk 1]
```diff
@@ -102,23 +98,28 @@ export default class Icon extends React.Component<
...
         this.setState((prevState) => ({...prevState, [attr]: null}))
       ),
-      notifyIconAction: () => ( this.props.onSelect
-        ? this.props.onSelect()
-        : null
-      ),
+      setContent: () => undefined,
+      registerInteractionHandler: () => undefined,
+      deregisterInteractionHandler: () => undefined,
+      notifyIconAction: () => (this.props.onSelect ? this.props.onSelect() : null),
```
Comment (moog16) [inline packages/text-field/icon/index.tsx]: just do `notifyIconAction: this.props.onSelect!`, and add a default prop for onSelect: `onSelect: () => {}`

File: packages/text-field/index.tsx [hunk 2]
```diff
@@ -321,32 +322,27 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
...
+          onKeyDown={() => foundation && foundation.handleTextFieldInteraction()}
+          key='text-field-container'>
+          {leadingIcon ? this.renderIcon(leadingIcon, onLeadingIconSelect) : null}
+          {foundation ? this.renderInput() : null}
+          {label && !fullWidth ? this.renderLabel() : null}
+          {outlined ? this.renderNotchedOutline() : null}
+          {!fullWidth && !textarea && !outlined ? this.renderLineRipple() : null}
+          {trailingIcon ? this.renderIcon(trailingIcon, onTrailingIconSelect) : null}
+        </div>
+        {(helperText || false) && <div className={cssClasses.HELPER_LINE}>
```
Comment (moog16) [inline packages/text-field/index.tsx]: can you just add the `HELPER_LINE` element and the helperText null check into the `renderHelperText` method. This block of JSX can then just be:

```
{this.renderHelperText()}
```

File: packages/text-field/index.tsx [hunk 3]
```diff
@@ -149,14 +154,16 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
...
-      'mdc-text-field--outlined': outlined,
-      'mdc-text-field--textarea': textarea,
+    return classnames(cssClasses.ROOT, Array.from(classList), className, {
+      [cssClasses.DENSE]: dense,
+      [cssClasses.DISABLED]: disabled,
+      [cssClasses.FOCUSED]: isFocused,
+      [cssClasses.INVALID]: isValid,
+      [cssClasses.OUTLINED]: outlined,
+      [cssClasses.TEXTAREA]: textarea,
       'mdc-text-field--fullwidth': fullWidth,
```
Comment (moog16) [inline packages/text-field/index.tsx]: can you also reference your PR in mdc web as a TODO?

File: packages/text-field/index.tsx [hunk 4]
```diff
@@ -221,24 +224,24 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
...
     return {
+      isFocused: () => this.state.isFocused,
       getNativeInput: () => {
-        let badInput;
-        let valid;
-        if (this.inputComponent_) {
-          badInput = this.inputComponent_.isBadInput();
-          valid = this.inputComponent_.isValid();
+        const component = this.inputComponent_;
+        if (component) {
```
Comment (moog16) [inline packages/text-field/index.tsx]: change this if/else to early exit:
```
if (!component) {
  return null;
}
return {...}
```

File: packages/text-field/icon/index.tsx [hunk 5]
```diff
@@ -21,7 +21,10 @@
 // THE SOFTWARE.
 import * as React from 'react';
 import classnames from 'classnames';
-import {MDCTextFieldIconFoundation} from '@material/textfield';
+import {
+  MDCTextFieldIconAdapter,
+  MDCTextFieldIconFoundation,
+} from '@material/textfield';
```
[the remark this answers is not in the document]
Comment (TroyTae) [reply packages/text-field/icon/index.tsx]: Can you tell me reason why follow this pattern?
Maybe tree shaking?

File: packages/text-field/icon/index.tsx [hunk 6]
```diff
@@ -57,16 +57,14 @@ export default class Icon extends React.Component<
...
 
   componentDidMount() {
-    this.foundation_ = new MDCTextFieldIconFoundation(this.adapter);
-    this.foundation_.init();
-    if (this.props.disabled) {
-      this.foundation_.setDisabled(true);
-    }
+    this.foundation = new MDCTextFieldIconFoundation(this.adapter);
+    this.foundation.init();
+    this.foundation.setDisabled(!!this.props.disabled);
```
[the remark this answers is not in the document]
Comment (TroyTae) [reply packages/text-field/icon/index.tsx]: You're right, now there is no additional feature for "false".
But later I thought setDisabled could have a function for "false".
(For example, when "false", add class name 'no-disabled')
So I change like this!

File: packages/text-field/icon/index.tsx [hunk 7]
```diff
@@ -90,7 +86,7 @@ export default class Icon extends React.Component<
     return this.props.onSelect ? 0 : -1;
   }
 
-  get adapter() {
+  get adapter(): MDCTextFieldIconAdapter {
```
[the remark this answers is not in the document]
Comment (TroyTae) [reply packages/text-field/icon/index.tsx]: Can you check this again?
It looks I just added type for adapter...

File: packages/text-field/icon/index.tsx [hunk 8]
```diff
@@ -75,9 +73,7 @@ export default class Icon extends React.Component<
   }
 
   componentWillUnmount() {
-    if (this.foundation_) {
-      this.foundation_.destroy();
-    }
+    this.foundation.destroy();
```
[the remark this answers is not in the document]
Comment (TroyTae) [reply packages/text-field/icon/index.tsx]: I know componentWillUnmount is executed after componentDidMount.
Is null check needed?

File: packages/text-field/icon/index.tsx (hunk 1 above)
Comment (TroyTae) [reply packages/text-field/icon/index.tsx]: I reflected it, but one question!
Is there any way to prevent type error for defaultProps?

**Comment by TroyTae** [comment]
@moog16 
Ummm if you think my progress speed is slow, talk to me about that.
Then I'll close this PR.
Because I don't want to interrupt release schedule.

**Comment by bartiaco** [comment]
@TroyTae and @moog16 Thank you both for your work on this. We're eagerly awaiting these updates <3

File: packages/text-field/icon/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (moog16) [reply packages/text-field/icon/index.tsx]: Its actually more of a convention borrowed from mdc web. https://github.com/material-components/material-components-web/blob/master/docs/code/best_practices.md#foundation-import-statements-must-not-use-re-exported-modules. It has no perf gain, etc, but we should remain consistent.

File: packages/text-field/icon/index.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (moog16) [reply packages/text-field/icon/index.tsx]: This is the foundation method:
```
  setDisabled(disabled: boolean) {
    if (!this.savedTabIndex_) {
      return;
    }

    if (disabled) {
      this.adapter_.setAttr('tabindex', '-1');
      this.adapter_.removeAttr('role');
    } else {
      this.adapter_.setAttr('tabindex', this.savedTabIndex_);
      this.adapter_.setAttr('role', strings.ICON_ROLE);
    }
  }
```

So when disabled == `false`, it actually adds role and tabindex.

File: packages/text-field/icon/index.tsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (moog16) [reply packages/text-field/icon/index.tsx]: I realize that, and I'm trying to remember why we did that. I also might be getting confused with something. Since we're inconsistent with this, please keep this change.

File: packages/text-field/icon/index.tsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (moog16) [reply packages/text-field/icon/index.tsx]: Ah...weird...I thought you had removed this, sorry.

File: packages/text-field/icon/index.tsx (hunk 1 above)
Comment (moog16) [reply packages/text-field/icon/index.tsx]: Gotcha - it needs to be `undefined`.

**Comment by moog16** [comment]
@TroyTae what are you thoughts on character count? Should we open a new issue for that to address outside of this PR? Or do you want to tackle this?

**Comment by moog16** [comment]
@TroyTae - you're doing great! The release schedule (as well as myself) is behind anyways.

File: packages/text-field/icon/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (TroyTae) [reply packages/text-field/icon/index.tsx]: I reflect it!
But OutlineAdapter and RippleAdapter had not their directory, so I keep like this:
```
import {
  MDCTextFieldOutlineAdapter,
  MDCTextFieldLineRippleAdapter,
} from '@material/textfield';
```

File: packages/text-field/icon/index.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (TroyTae) [reply packages/text-field/icon/index.tsx]: Oh I see, thank you!

**Comment by TroyTae** [comment]
@moog16 
I consider character count and I already define it in props.
And I was planning make it after update version.
But the counter can work separately, so if someone want to do it, it can make a separate PR.

**Comment by TroyTae** [comment]
@moog16 
I consider character count and I already define it in props.
But I think this PR is too heavy.
How about verification and merge this PR's changes, and then create another PR?

File: packages/text-field/icon/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (moog16) [reply packages/text-field/icon/index.tsx]: ah interesting...do you mind changing that too?

**Comment by moog16** [comment]
@TroyTae I'm thinking the same thing. Let's work on it separately.

File: packages/text-field/index.tsx [hunk 9]
```diff
@@ -329,23 +315,23 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
         <div
           {...this.otherProps}
           className={this.classes}
-          onClick={() => foundation && foundation.handleTextFieldInteraction()}
-          onKeyDown={() => foundation && foundation.handleTextFieldInteraction()}
+          onClick={() => foundation!.handleTextFieldInteraction()}
```
Comment (moog16) [inline packages/text-field/index.tsx]: can you change this to `onClick={foundation. handleTextFieldInteraction}`? Same with onKeyDown handler

Comment (TroyTae) [reply packages/text-field/index.tsx]: Can you tell me what is this syntax mean?
`onClick={foundation. handleTextFieldInteraction}`
Causes an error!

Comment (moog16) [reply packages/text-field/index.tsx]: foundation.handleTextFieldInteraction is a function. So the onClick event will trigger function.

File: packages/text-field/index.tsx [hunk 10]
```diff
@@ -209,41 +213,33 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
...
+      isFocused: () => this.state.isFocused,
       getNativeInput: () => {
-        let badInput;
-        let valid;
-        if (this.inputComponent_) {
-          badInput = this.inputComponent_.isBadInput();
-          valid = this.inputComponent_.isValid();
+        const component = this.inputComponent_;
+        if (!component) {
+          return null;
```
Comment (moog16) [inline packages/text-field/index.tsx]: i think we should actually throw an error here.

File: packages/text-field/index.tsx [hunk 11]
```diff
@@ -276,22 +272,13 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
...
-  }
-
   inputProps(child: React.ReactElement<InputProps<T>>) {
     // ref does exist on React.ReactElement<InputProps<T>>
     // @ts-ignore
     const {props, ref} = child;
     return Object.assign({}, props, {
       foundation: this.state.foundation,
       handleFocusChange: (isFocused: boolean) => this.setState({isFocused}),
-      handleValueChange: (value: string | number, cb: () => void) => this.setState({value}, cb),
```
Comment (moog16) [inline packages/text-field/index.tsx]: why was this removed?

**Comment by moog16** [comment]
@TroyTae the screenshot tests aren't working. Add to the `test/screenshot/text-field/index.tsx` file ```import '../../../packages/text-field/index.scss`

File: packages/text-field/index.tsx (hunk 9 above)
Comment (TroyTae) [reply packages/text-field/index.tsx]: Ah I see, I confused because of space before "handleTextFieldInteraction".
I reflect it!

File: packages/text-field/index.tsx (hunk 11 above)
Comment (TroyTae) [reply packages/text-field/index.tsx]: Input.tsx already has this method.
And I cannot find any special purpose of this function.
So I remove it.

File: packages/text-field/index.tsx [hunk 12]
```diff
@@ -219,7 +219,7 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
       getNativeInput: () => {
         const component = this.inputComponent_;
         if (!component) {
-          return null;
+          throw Error('The input did not render properly');
```
Comment (moog16) [inline packages/text-field/index.tsx]: Prepend `MDCReactTextField:The input...`

File: packages/text-field/index.tsx [hunk 13]
```diff
@@ -320,32 +308,25 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
...
-        </React.Fragment>
-      );
-    }
-    return textField;
+    return (
+      <React.Fragment>
+        <div
+          {...this.otherProps}
+          className={this.classes}
+          onClick={() => foundation!.handleTextFieldInteraction()}
```
Comment (moog16) [inline packages/text-field/index.tsx]: Looks like you changed this back. Can it not be `onClick={foundation!.handle...}`?

File: packages/text-field/package.json [hunk 14]
```diff
@@ -20,7 +20,7 @@
     "@material/react-floating-label": "^0.11.0",
     "@material/react-line-ripple": "^0.11.0",
     "@material/react-notched-outline": "^0.11.0",
-    "@material/textfield": "^0.41.0",
+    "@material/textfield": "^1.1.1",
```
Comment (moog16) [inline packages/text-field/package.json]: I actually think you can remove this.

File: packages/text-field/index.tsx (hunk 13 above)
Comment (TroyTae) [reply packages/text-field/index.tsx]: Yes we need to use arrow function for "this" keyword.

**Comment by TroyTae** [comment]
@moog16 
I read this https://github.com/material-components/material-components-web-react/blob/master/docs/screenshot-tests.md.
And try to update golden.json but I keep getting JSON related errors.
Can you help me?

**Comment by moog16** [comment]
@TroyTae can you tell me what JSON errors you're getting?

**Comment by TroyTae** [comment]
@moog16 
This is my error.
I can see this when I run `npm run capture` or `npm run image:diff`.
```
pptruser@f7181152fc75:~/material-components-web-react$ npm run capture

> @ capture /home/pptruser/material-components-web-react
> MDC_COMMIT_HASH=$(git rev-parse --short HEAD) MDC_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD) mocha --compilers js:babel-core/register --ui tdd --timeout 90000 test/screenshot/capture-suite.js

undefined:1
undefined
^

SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at Object.<anonymous> (/home/pptruser/material-components-web-react/test/screenshot/screenshot.js:26:21)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at loader (/home/pptruser/material-components-web-react/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/pptruser/material-components-web-react/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/pptruser/material-components-web-react/test/screenshot/capture-suite.js:1:1)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at loader (/home/pptruser/material-components-web-react/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/pptruser/material-components-web-react/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at /home/pptruser/material-components-web-react/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/home/pptruser/material-components-web-react/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/home/pptruser/material-components-web-react/node_modules/mocha/lib/mocha.js:536:10)
    at Object.<anonymous> (/home/pptruser/material-components-web-react/node_modules/mocha/bin/_mocha:582:18)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
    at startup (internal/bootstrap/node.js:285:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)
```

**Comment by moog16** [comment]
@TroyTae in your screenshot/screenshot.tsx file there is:

```
const defaultMetadata = {
  commit: commitHash,
  branch: branchName,
};```

Something is being set correctly. I think one of those is undefined. I think since you're developing in your own branch it could be that the `branchName` is not set correctly. Could you try to console.log both of those while running your tests?

**Comment by TroyTae** [comment]
@moog16 
I couldn't figure out why.
But it's fine after removing all of docker images.

**Comment by moog16** [comment]
@TroyTae we shouldn't have changed all of those goldens. Can you revert the ones that aren't related to text field?

File: packages/text-field/index.tsx [hunk 15]
```diff
@@ -320,32 +308,25 @@ class TextField<T extends HTMLElement = HTMLInputElement> extends React.Componen
...
+          onKeyDown={() => foundation!.handleTextFieldInteraction()}
+          key='text-field-container'>
+          {leadingIcon ? this.renderIcon(leadingIcon, onLeadingIconSelect) : null}
+          {this.renderInput()}
+          {label && !fullWidth ? this.renderLabel() : null}
```
Comment (moog16) [inline packages/text-field/index.tsx]: The label should be rendered inside the notched outline if its `outlined`. See https://github.com/material-components/material-components-web-react/pull/816

Comment (TroyTae) [reply packages/text-field/index.tsx]: Reflect it!

Also I change adding outlined class name when text-field is not fullWidth.
(https://github.com/material-components/material-components-web/tree/master/packages/mdc-textfield#full-width)
Confirm this please :)

Comment (moog16) [reply packages/text-field/index.tsx]: looks good!

**Comment by moog16** [comment]
@TroyTae Everything looks good in the PR. I can't get the screenshot tests up and running, so that might be why the screenshot tests aren't passing.

**Comment by TroyTae** [comment]
Work on #817 after this PR merged.

**Comment by moog16** [comment]
@TroyTae tests still failing...Also travis looks like it is stuck. Can you try restarting?

**Comment by moog16** [comment]
@TroyTae please apply changes to package.json and packages/text-field/index.tsx:

https://github.com/material-components/material-components-web-react/commit/958fc9ab90a18c025e46aae43dbea9615a76127c

File: test/screenshot/golden.json [hunk 16]
```diff
@@ -27,7 +27,7 @@
   "text-field/textArea": "b4c009f5f7637f7103380a2dd93bd6387d1ccc22d031323e6f0472949ce35881",
   "text-field/standard": "61cf0ebade2a09263d3a015c26cde28b3b3e67ab9d9bb4c494ac4823b9e8000b",
   "text-field/fullWidth": "26fa1e96054939384efb6427d93967bdbbc05ecc00bf7e4f13ab17cbe3e367fb",
-  "text-field/outlined": "91e95a9bfb4e3f75ba9bb6a7ccf6a379b944d4960aaffd4ca2e4026a3f3daa71",
+  "text-field/outlined": "3e1edca4eb2e215ebff69729afd44a18b5b8796d2754d7eba0451d8b5bc1619a",
```
Comment (moog16) [inline test/screenshot/golden.json]: This needs to be updated. I'm working on it

File: test/screenshot/golden.json [hunk 17]
```diff
@@ -24,10 +24,11 @@
   "tab-scroller": "468866dd0c222b36b55485ab44a5760133a4ddfb2a6cf81e6ae4672d7e02a447",
   "text-field/helper-text": "59604d0f39e0846fc97aae7573d317dded215282a677e4641c5e33426e3a2a1e",
   "text-field/icon": "0bbc8c762d27071e55983e5742548d166864b6fcebc0b9f1e413523fb93b7075",
-  "text-field/textArea": "b4c009f5f7637f7103380a2dd93bd6387d1ccc22d031323e6f0472949ce35881",
-  "text-field/standard": "61cf0ebade2a09263d3a015c26cde28b3b3e67ab9d9bb4c494ac4823b9e8000b",
-  "text-field/fullWidth": "26fa1e96054939384efb6427d93967bdbbc05ecc00bf7e4f13ab17cbe3e367fb",
-  "text-field/refTest": "742fe55ba0f3ca11c74beef5ea9737e2eaec37d9c8524552f3b06c6cb25f4157",
+  "text-field/textArea": "c52dd8a89f22c44f4a832d467f3df5568c77075e26576664de31538428387734",
```
Comment (moog16) [inline test/screenshot/golden.json]: i updated this...

Comment (TroyTae) [reply test/screenshot/golden.json]: Ah thank you!
But screenshot test is failing from your second merge.
And branch test is still stuck.
Is there something that I work in here additionally?

**Comment by moog16** [comment]
@TroyTae please see comment https://github.com/material-components/material-components-web-react/pull/812#issuecomment-487751687

**Comment by moog16** [comment]
i tried updating the branches - maybe this will kick off the Travis CI. https://github.com/travis-ci/travis-ci/issues/10204

**Comment by TroyTae** [comment]
Maybe we had same issue in here (https://github.com/material-components/material-components-web-react/pull/804#issuecomment-482480304)

**Comment by moog16** [comment]
Yes that's the comment...I'm going to try a different combination

**Comment by moog16** [comment]
@TroyTae please add `'outlined'` to the text-field/variants.ts file

**Comment by TroyTae** [comment]
@moog16 
Oh you already work...

**Comment by moog16** [comment]
Yup! Figured it out :) 

I'm trying to get the tests passing...I think everything looks good.

**Comment by moog16** [comment]
@TroyTae are you on linux? Have you tried running `npm start`?

File: packages/text-field/package.json [hunk 18]
```diff
@@ -20,7 +20,6 @@
     "@material/react-floating-label": "^0.11.0",
     "@material/react-line-ripple": "^0.11.0",
     "@material/react-notched-outline": "^0.11.0",
-    "@material/textfield": "^0.41.0",
```
Comment (moog16) [inline packages/text-field/package.json]: this is the issue!

**Comment by moog16** [comment]
@TroyTae - I fixed it all, but I merged rc0.12.0 --> feat/mdcweb-typescript-update. I think the best course of action is to cherry pick all of our commits into a new branch. And create a new PR. What are your thoughts?

File: packages/text-field/package.json (hunk 18 above)
Comment (TroyTae) [reply packages/text-field/package.json]: What is the detail causes?
I remember that I deleted this code in your review...

**Comment by TroyTae** [comment]
@moog16 
We need to merge every changes in branch rc 0.12.0 right?
I'll try that

**Comment by TroyTae** [comment]
I signed it!

**Comment by TroyTae** [comment]
@moog16 
I think Cherry Peek is the most obvious way.
But It takes many times.
So I used rebase and merge.

**Comment by moog16** [comment]
Great!! What happened to tests?

**Comment by TroyTae** [comment]
@moog16 
Ummm can you tell me what do you mean?
I don't touch test code.
I just solve conflict!

File: packages/text-field/package.json (hunk 18 above)
Comment (moog16) [reply packages/text-field/package.json]: hmm no worries :) all good now.

**Comment by moog16** [comment]
@TroyTae thanks for fixing the PR...looking at textArea, which is still failing for some reason.

**Comment by moog16** [comment]
@TroyTae please add 'outlined' to test/screenshot/text-field/variants.tsx

**Comment by moog16** [comment]
thanks!

**Comment by TroyTae** [comment]
And maybe I need to add class 'mdc-text-field--no-label' (https://github.com/material-components/material-components-web/tree/816139c1a10647938f345ab7d40b94b3e2ce91ed/packages/mdc-textfield#text-field-without-label).
I working now.

**Comment by moog16** [comment]
@TroyTae Thanks for fixing and all your help. That was a long one. Ball is in my court now. :)

**Comment by TroyTae** [comment]
@moog16 
Thank you for all your help!
I'm looking forward this release :)
